graylog2-web-interface/src/components/sidecars/sidecars/VerboseMessageModal.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import lodash from 'lodash';
+
+import BootstrapModalWrapper from 'components/bootstrap/BootstrapModalWrapper';
+import CombinedProvider from 'injection/CombinedProvider';
```

Comment:
Once the next variable is removed, this import will not be used and can be removed as well.

```suggestion
```